### PR TITLE
Update error message + docs when adding local users

### LINKED ISSUE #3083: Provide helpful error message when adding local users (teleport - enterprise)
**What happened:** User was trying to create a user for a k8s group with the following command

`sudo tctl users add joe --k8s-groups="system:masters"`

Error recieved was:

`error: unknown long flag '--k8s-groups'`

**What you expect to happen:**

Whenever we add a local user in enterprise, and a user tries to use `--k8s-groups` or `--logins` error handling should provide a helpful error message instructing the user to use RBAC and create a role with k8s groups. When creating a local user, simply use `--roles` flag. 

Also update documentation [here](https://gravitational.com/teleport/docs/admin-guide/#kubernetesteleport-configuration) and add note mentioning to create a k8s group.

**References:**

Add creates a new sign-up token and prints token URL to stdout [here](https://github.com/gravitational/teleport/blob/master/tool/tctl/common/user_command.go#L105)

Function `TryRun` takes CLI command as an argument and executes it [here](https://github.com/gravitational/teleport/blob/master/tool/tctl/common/user_command.go#L87)

Function `ParseRoles` takes list of roles and returns a slice of roles, or an error if parsing failed [here](https://github.com/gravitational/teleport/blob/master/roles.go#L75)

**Related issue:** #2950

### PR DESCRIPTION
**Pull request by aelkugia**
When adding local users  error handling wasn't clear reporting `error: unknown long flag ...`. This PR updates the docs + adds more context to error

Resolves: #3083

### CODE REVIEW

File: docs/4.2/admin-guide.md [hunk 1]
```diff
@@ -2047,6 +2047,10 @@ belong to:
 $ tctl users add joe --k8s-groups="system:masters"
 ```
 
+!!! tip "NOTE"
+
+    The above command is for the Teleport OSS version. Teleport Enterprise customers manage `k8s-groups` through RBAC.
```
Comment (benarent) [inline docs/4.2/admin-guide.md]: @aelkugia Could you add a sample role to https://github.com/gravitational/teleport/tree/master/examples/resources

Comment (aelkugia) [reply docs/4.2/admin-guide.md]: Added sample k8s-groups role

**Review by benarent — APPROVED** [review]

**Review by russjones — APPROVED** [review]

**Comment by russjones** [comment]
@gravitational-jenkins retest this please

**Comment by russjones** [comment]
@gravitational-jenkins retest this please

**Comment by russjones** [comment]
@gravitational-jenkins retest this please

**Comment by russjones** [comment]
@gravitational-jenkins retest this please

**Comment by russjones** [comment]
@gravitational-jenkins retest this please

**Pull request merged**